x-pack/platform/plugins/shared/fleet/server/services/epm/elasticsearch/template/remove.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import pMap from 'p-map';
+import type { ElasticsearchClient } from '@kbn/core/server';
+
+import { FleetError } from '../../../../errors';
+import {
+  MAX_CONCURRENT_COMPONENT_TEMPLATES,
+  USER_SETTINGS_TEMPLATE_SUFFIX,
+} from '../../../../constants';
+import { appContextService } from '../../..';
+
+export const deleteComponentTemplates = async (
+  esClient: ElasticsearchClient,
+  componentTemplateIds: string[]
+) => {
+  const logger = appContextService.getLogger();
+  if (componentTemplateIds.length) {
+    logger.info(`Deleting currently installed component template ids ${componentTemplateIds}`);
```

Comment:
I think this is missing a map inside the string to list all the template Ids.

Reply:
Since its an array of strings, this will print all the string values as is without needing to map it out